tests/integration/commonjs.test.js

```diff
@@ -1,12 +1,21 @@
 import { expect } from 'chai';
 
 describe('CommonJS', () => {
-  it('should works as expected', () => {
+  it('should work as expected with legacy default export', () => {
     const jsf = require('../../dist/main.cjs.js').default;
 
     expect(jsf).not.to.be.undefined;
     expect(jsf.VERSION).not.to.be.undefined;
     expect(jsf.generate).not.to.be.undefined;
     expect(jsf.generate({ const: 42 })).to.eql(42);
   });
+
+  it('should work as expected with named export', () => {
+    const { JSONSchemaFaker } = require('../../dist/main.cjs.js').default;
```

Comment:
I think you're extracting `JSONSchemaFaker` from the default export, just remove the `.default` part.

Reply:
Oops! Yeah, that doesn't help, but there's still another problem upstream. Looking into it now.